vendor/github.com/skytap/skytap-sdk-go/skytap/vm.go

```diff
@@ -458,7 +434,7 @@ func (s *VMsServiceClient) changeRunstate(ctx context.Context, environmentID str
 	}
 
 	var updatedVM VM
-	_, err = s.client.do(ctx, requestCreate, &updatedVM)
+	_, err = s.client.do(ctx, requestCreate, &updatedVM, vmRunStateNotBusy(environmentID, id), opts)
```

Comment:
This needs to check that the environment is not busy. When a VM is busy, its environment is also busy, so attempting to set runstate on other VMs will fail with 423 Locked.

Reply:
Ok - point taken. For correctness I will change it. The resulting behaviour however will still be the same - the API will retry on 422 or 423.